.readthedocs.yml

```diff
@@ -11,7 +11,7 @@ submodules:
   recursive: true
 
 build:
-  os: ubuntu-22.04
+  os: ubuntu-24.04
```

Comment:
This seem to be a separate thing..

Reply:
It's only purpose is to fix installing the new package.